src/state.rs

```diff
@@ -76,6 +85,53 @@ impl State {
         Ok(state)
     }
 
+    pub fn get_votes<BS: Blockstore>(
+        &self,
+        store: &BS,
+        cid: &Cid,
+    ) -> Result<Option<Votes>, ActorError> {
+        let hamt = self
+            .window_checks
+            .load(store)
+            .map_err(|_| actor_error!(illegal_state, "cannot load votes hamt"))?;
+        let amount = hamt
```

Comment:
```suggestion
        let votes = hamt
```
maybe you copy-pasted and forgot to update?